src/styles/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  justify-content: space-between;
+
+  &__logo-link {
+    line-height: 0;
+  }
+
+  &__list{
+    display: flex;
+    margin: 0;
+    padding: 0;
```

Comment:
you can add these reset styles to your reset.scss file

```
ul {
  margin: 0;
  padding: 0;
  list-style: none;
}

a {
  text-decoration: none;
}
````

after delete unnecessary styles from your scss blocks